earthfile2llb/earthfile2llb.go

```diff
@@ -77,6 +77,10 @@ type ConvertOpt struct {
 	// Parallelism is a semaphore controlling the maximum parallelism.
 	Parallelism *semaphore.Weighted
 
+	// AvailableSecrets is a map to indicate of secrets are available within the attachables
+	// this is used to include better error messages when a secret is missing
+	AvailableSecrets map[string]interface{}
```

Comment:
Why `interface{}` over `struct{}`, especially since you populate it with empty struct?

Reply:
good point, i'll change that.